src/Modules/ContentBlocks/Domain/ContentBlock/ContentBlockRepository.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace ForkCMS\Modules\ContentBlocks\Domain\ContentBlock;
+
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+use ForkCMS\Modules\Frontend\Domain\Block\Block;
+use ForkCMS\Modules\Frontend\Domain\Block\BlockRepository;
+use ForkCMS\Modules\Internationalisation\Domain\Locale\Locale;
+
+/**
+ * @method ContentBlock|null find($id, $lockMode = null, $lockVersion = null)
+ * @method ContentBlock|null findOneBy(array $criteria, array $orderBy = null)
+ * @method ContentBlock[]    findAll()
+ * @method ContentBlock[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+final class ContentBlockRepository extends ServiceEntityRepository
+{
+    public function __construct(
+        ManagerRegistry $managerRegistry,
+        private readonly BlockRepository $blockRepository
+    ) {
+        parent::__construct($managerRegistry, ContentBlock::class);
+    }
+
+    public function save(ContentBlock $contentBlock): void
+    {
+        $entityManager = $this->getEntityManager();
+        $entityManager->persist($contentBlock);
+        $entityManager->flush();
+
+        if ($contentBlock->getStatus() !== Status::Archived) {
+            $this->updateWidget($contentBlock);
+        }
+    }
+
+    public function remove(ContentBlock $contentBlock): void
+    {
+        $entityManager = $this->getEntityManager();
+        $entityManager->remove($contentBlock);
+        $entityManager->flush();
+    }
+
+    public function removeMultiple(array $contentBlocks): void
```

Comment:
are you using this somewhere?
If not you can remove this. if you are using it you need to add the phpdoc for the array or work with ...ContentBlock $contentBlocks

Reply:
phpdoc added: https://github.com/forkcms/forkcms/pull/3555/commits/0eee4916220942e1607775ea492552d6a300601a